webui/react/src/pages/F_ExpList/glide-table/MultiSortMenu.tsx

```diff
@@ -12,7 +12,7 @@ import { Loadable } from 'utils/loadable';
 import css from './MultiSortMenu.module.scss';
 
 // in the list of columns from the api but not supported by the sort functionality
-const BANNED_COLUMNS = new Set(['tags']);
+const BANNED_COLUMNS = new Set(['tags', 'searcherMetric', 'searcherMetricValue']);
```

Comment:
Why do we not support sorting by searcher metric value any more?

Reply:
oh, I missed that `searcherMetricsVal` is available in the server-side `sortExperiments` (but not `searcherMetric`).
I changed the name in this code of the new glide table column to `searcherMetricsVal`, so it matches.

I added some code for when a column in the settings is no longer available.

Filtering `searcherMetricsVal` doesn't work, because it's treated as a string and could be filtering numbers. I added a new BANNED_FILTER_COLUMNS to avoid showing filtering on that column menu.